config/config.go

```diff
@@ -322,18 +330,25 @@ func loadReceivers(v *viper.Viper, factories map[string]receiver.Factory) (confi
 		receiverCfg.SetType(typeStr)
 		receiverCfg.SetName(fullName)
 
-		// Now that the default config struct is created we can Unmarshal into it
-		// and it will apply user-defined config on top of the default.
-		customUnmarshaler := factory.CustomUnmarshaler()
-		if customUnmarshaler != nil {
-			// This configuration requires a custom unmarshaler, use it.
-			err = customUnmarshaler(subViper, key, receiverCfg)
-		} else {
-			// Standard viper unmarshaler is fine.
-			// TODO(ccaraman): UnmarshallExact should be used to catch erroneous config entries.
-			// 	This leads to quickly identifying config values that are not supported and reduce confusion for
-			// 	users.
-			err = subViper.UnmarshalKey(key, receiverCfg)
+		// Unmarsh only the subconfig for this receiver.
```

Comment:
nit: copy and paste of typo above (we will skip same below)
```suggestion
		// Unmarshal only the subconfig for this receiver.
```

Reply:
Done.